### PR TITLE
New transformer: SecondsToString.

### PR DESCRIPTION
**Pull request by tundisto**
Submitting new transformer SecondsToString.  Takes a number of seconds as input and outputs the number of days, hours, minutes, and seconds it translates to in the format "Nd Nh Nm Ns"

## Status
Ready

## Related Issues
N/A

## Description
Code + unit test

## Screenshots
Paste here any images that will help the reviewer

## Related PRs
N/A

## Required version of Demisto
4.x

## Does it break backward compatibility?
   - No

## Must have
- [X] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Additional changes
Describe additional changes done, for example adding a function to common server.

## Technical writer review
@kirbles19 please review the following:
- [ ] [YAML file](https://github.com/demisto/content/blob/6ec32e78d5c5f7313acbc90f3f75d7318ca54ca8/Scripts/SecondsToString/SecondsToString.yml)

### CODE REVIEW

**Comment by kirbles19** [comment]
@Itay4 Done.

File: Scripts/SecondsToString/SecondsToString.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+commonfields:
+  id: SecondsToString
+  version: -1
+name: SecondsToString
+script: ''
+type: python
+tags:
+- transformer
+- date
```
Comment (Itay4) [inline Scripts/SecondsToString/SecondsToString.yml]: i think this should be number, no?

File: Scripts/SecondsToString/SecondsToString.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+def display_time(seconds):
+    result = []
+
+    for name, count in intervals:
+        value = seconds // count
+        if value:
+            seconds -= value * count
+            result.append("{}{}".format(value, name))
```
Comment (Itay4) [inline Scripts/SecondsToString/SecondsToString.py]:
```suggestion
            result.append(f'{value}{name}')
```

File: Scripts/SecondsToString/SecondsToString.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    return ' '.join(result)
+
+
+def main(value):
+    unformatted_time = int(value)
+    formatted_time = display_time(unformatted_time)
+    return formatted_time
+
+
+if __name__ == "__builtin__" or __name__ == "builtins":
```
Comment (Itay4) [inline Scripts/SecondsToString/SecondsToString.py]:
```suggestion
if __name__ in ('__builtin__', 'builtins'):
```

File: Scripts/SecondsToString/SecondsToString.yml (hunk 1 above)
Comment (tundisto) [reply Scripts/SecondsToString/SecondsToString.yml]: I don't think so.  It's for converting a date-type field so it belongs in the date section.  The fact that the input is actually a number is incidental as it's just an epoch timestamp.

Comment (tundisto) [reply Scripts/SecondsToString/SecondsToString.yml]: Sorry, my reply was rubbish -- wasn't thinking properly, not an epoch date at all.  I do think it still belongs in the date section, though, as it relates to time.  But at the end of the day, I don't really care.

**Comment by tundisto** [comment]
@Itay4 please feel free to merge -- I'm unable to

**Pull request merged**